frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -162,6 +178,23 @@ const deleteAppAdmin = async (admin: FamAppAdminGetResponse) => {
                         @deleteUserRoleAssignment="deleteUserRoleAssignment"
                     />
                 </TabPanel>
+
+                <TabPanel
+                    v-if="LoginUserState.hasAccess(APP_ADMIN_ROLE)"
```

Comment:
here as well, we don't have delegated admin for FAM, so ` && selectedApplicationId !== FAM_APPLICATION_ID`

Reply:
I'll make a refactor